src/docs/get-started/test-drive/_vscode.md

```diff
@@ -60,6 +60,7 @@ contains a simple demo app that uses [Material Components][].
 [Quickly switching between Flutter devices]: https://dartcode.org/docs/quickly-switching-between-flutter-devices
 [status bar]: {% asset tools/vs-code/device_status_bar.png @path %}
 
+</div>
```

Comment:
Did you want to move down the other reference links that are still above this <div> ?

Reply:
Done.